locations/ak36.yml

```diff
@@ -77,70 +74,63 @@ mgmt:
     # hypervisor: 9       # .137 (strange)
     ak36-philmel60: 10    # .138
     ak36-teufelsberg: 11  # .139
-    # ...
+    # services
+    config: 22            # .150
+    monitor: 23           # .151
     uisp: 30              # .158
 
 # Mesh Network: 10.31.130.160/27
 # Not listed here: strom-gw 10.31.130.190/32
 mesh_links:
   - name: mesh_philmel
-    ifname: eth2
+    ifname: eth1.10
     ipv4: 10.31.130.160/32
     ipv6: 2001:bf7:750:4001::1/128
     metric: 1024
     ptp: true
 
   - name: mesh_flughafen
-    ifname: eth3
+    ifname: eth1.11
     ipv4: 10.31.130.161/32
     ipv6: 2001:bf7:750:4001::2/128
     metric: 1024
     ptp: true
 
   - name: mesh_dtmb
-    ifname: eth4
+    ifname: eth1.12
     ipv4: 10.31.130.162/32
     ipv6: 2001:bf7:750:4001::3/128
     metric: 1024
     ptp: true
 
   - name: mesh_friedenau
-    ifname: eth5
+    ifname: eth1.13
     ipv4: 10.31.130.163/32
     ipv6: 2001:bf7:750:4001::4/128
     metric: 1024
     ptp: true
 
-  - name: mesh_backbone
-    ifname: eth6
+  - name: mesh_bbbvpn
+    ifname: eth1.198
     ipv4: 10.31.130.164/32
     ipv6: 2001:bf7:750:4001::5/128
```

Comment:
please make IPv6 a comment as we've done with https://github.com/freifunk-berlin/bbb-configs/blob/master/locations/strom.yml#L82 so no babeld config will be generated for this interface